examples/Console/opentelemetry-collector-otlp-example.yaml

```diff
@@ -0,0 +1,14 @@
+receivers:
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/examples/Console/TestPrometheusExporter.cs#L38
Instead of checking in this file, we could make it as a comment in the example.

Reply:
Committing the file allows users can just copy the command from the comment and it will run successfully. If you think it is confusing I can put the file in a dedicated folder.

Reply:
Agree - can you put it in a separate folder and adjust docker command?

Reply:
Done.